dataworks-admin-utils.tf

```diff
@@ -0,0 +1,34 @@
+resource "github_repository" "dataworks-admin-utils" {
+  name        = "dataworks-admin-utils"
+  description = "Contains dataworks administrative utilities"
```

Comment:
Should be `DataWorks`
comment not request

Reply:
Done, ta :-)